src/nimodinst/metadata/config_addon.py

```diff
@@ -1,4 +1,5 @@
 # We need to maintain the version here since it needs to be updated by the build process on GitHub
 config_additional_config = {
     'module_version': '1.1.1.dev0',
+    'last_tested_version': '18.0.0',
```

Comment:
The name of this key is terrible. It sounds like the last version on Python module `nimodinst` that we tested is 18.0.0.

I think we should take this opportunity to rename this key to something that isn't confusing like `latest_runtime_version_tested_against`

Reply:
Changed key to `latest_runtime_version_tested_against`